src/pages/about.js

```diff
@@ -21,49 +21,38 @@ class App extends React.Component {
           <p>
             Blacks Who Engineer is a Twitter directory of
             accomplished Black folks in the
-            design industry. It aims to help people find notable and relevant
+            software industry. It aims to help people find notable and relevant
```

Comment:
Regarding our Slack conversation about possibly expanding this directory to include other types of engineers, I wonder if it would make sense to change this from `software industry` to `tech industry`?

Reply:
Sounds good, updated everywhere that we use this sentence, too